cmake/modules/FindLibDvd.cmake

```diff
@@ -202,18 +202,81 @@ if(NOT WIN32)
     set(LIBDVD_FOUND 1 CACHE BOOL "libdvd found" FORCE)
     endif()
 else()
-  # Dynamically loaded on Windows
-  find_path(LIBDVD_INCLUDE_DIR dvdcss/dvdcss.h)
-
+  # windows specific builds
+  find_program(PATCH patch)
   include(FindPackageHandleStandardArgs)
-  find_package_handle_standard_args(LibDvd REQUIRED_VARS LIBDVD_INCLUDE_DIR)
+  find_package_handle_standard_args(PATCH REQUIRED_VARS PATCH)
 
-  if(LIBDVD_FOUND)
-    set(LIBDVD_INCLUDE_DIRS ${LIBDVD_INCLUDE_DIR})
+  set(INSTALL_PREFIX ${CMAKE_BINARY_DIR}/${CORE_BUILD_DIR}/libdvd)
+  set(DEPENDS_TARGETS_DIR ${CMAKE_SOURCE_DIR}/tools/depends/target)
+  set(dvdlibs libdvdcss libdvdread libdvdnav)
 
-    add_custom_target(dvdnav)
-    set_target_properties(dvdnav PROPERTIES FOLDER "External Projects")
+  foreach(dvdlib ${dvdlibs})
+    file(GLOB VERSION_FILE ${CMAKE_SOURCE_DIR}/tools/depends/target/${dvdlib}/DVD*-VERSION)
+    file(STRINGS ${VERSION_FILE} VER)
+    string(REGEX MATCH "VERSION=[^ ]*$.*" ${dvdlib}_VER "${VER}")
+    list(GET ${dvdlib}_VER 0 ${dvdlib}_VER)
+    string(SUBSTRING "${${dvdlib}_VER}" 8 -1 ${dvdlib}_VER)
+    string(REGEX MATCH "BASE_URL=([^ ]*)" ${dvdlib}_BASE_URL "${VER}")
+    list(GET ${dvdlib}_BASE_URL 0 ${dvdlib}_BASE_URL)
+    string(SUBSTRING "${${dvdlib}_BASE_URL}" 9 -1 ${dvdlib}_BASE_URL)
+    string(TOUPPER ${dvdlib} DVDLIB)
+    set(${DVDLIB}_URL ${${dvdlib}_BASE_URL}/archive/${${dvdlib}_VER}.tar.gz)
+  endforeach()
```

Comment:
please don't duplicated this. 
The whole extra block for win shouldn't be needed and instead merged with non-depends build above.

Reply:
I did not get your thought

Reply:
Above there is already the logic for building the libdvd libs for posix systems not using our depends.
You should integrate it there and only difference where it is really necessary (autoconf vs cmake, maybe also something else)

Reply:
done